webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/services/GitUtils.scala

```diff
@@ -51,9 +51,9 @@ trait GitRepositoryProvider {
   /**
    * Obtain access to JGit porcelain API.
    */
-  def git: IOResult[Git]
+  def git: Git
```

Comment:
This is the important part of the change. 
We don't want to create a Git object each time, just have it ready. The creation should only happen one time at boot.